packages/light-client/src/spec/validateLightClientUpdate.ts

```diff
@@ -113,7 +113,7 @@ export function validateLightClientUpdate(
 
   const signingRoot = ssz.phase0.SigningData.hashTreeRoot({
     objectRoot: ssz.phase0.BeaconBlockHeader.hashTreeRoot(update.attestedHeader.beacon),
-    domain: store.config.getDomain(update.signatureSlot, DOMAIN_SYNC_COMMITTEE),
+    domain: store.config.getDomain(update.signatureSlot - 1, DOMAIN_SYNC_COMMITTEE),
```

Comment:
or should we use attestedHeader.beacon.slot?

Reply:
ahh just checked the linked PR , lgtm :+1:

Reply:
sadly, attested_header.beacon.slot may be incorrect.

Example:

BELLATRIX_FORK_EPOCH = 0
CAPELLA_FORK_EPOCH = 255
DENEB_FORK_EPOCH = 256

attested_header.slot = 8159 (bellatrix)
signature_slot = 8192 (Deneb)
fork_version_slot = 8191 (Capella)

so, sync committee would be determined by sync committee at slot 8192, the fork version is determined by slot 8191, and the object is from slot 8159

Reply:
thanks @etan-status for the insightful example :pray:  :heart: